examples/complete-vpc/main.tf

```
@@ -208,7 +202,7 @@ data "aws_iam_policy_document" "dynamodb_endpoint_policy" {
       test     = "StringNotEquals"
       variable = "aws:sourceVpce"
 
-      values = [data.vpc.vpc_id]
+      values = [module.vpc.vpc_id]
```

oy, this is still wrong! it should be

```suggestion
      values = [data.aws_vpc_endpoint.dynamodb.id]
```

because the condition is "source VPC endpoint" not "source VPC"

---

That's what it was before #742 (attempted to) change it to the vpc id! Oi! Revert and revert and revert!

---

we can leave it as is now, but the condition is wrong - should be `"aws:SourceVpc"`

---

Looking at it again now, the data source *is* a race condition. The comment makes no sense. This whole thing is confusing to me. Why not just get the vpce id from an output of the vpc endpoint module?